presto-docs/src/main/sphinx/release/release-0.248.1.rst

```diff
@@ -0,0 +1,8 @@
+===============
+Release 0.248.1
+===============
+
+General Changes
+------------
+* Remove optimization to avoid planning unnecessary LIMIT/TopN/Sort/DistinctLimit when relation is know to single row or less rows than requested.
```

Comment:
That was the original PR title. Should I put the whole PR title into a quote?

Reply:
We should not be putting the whole PR title. Maybe - we can do this - this was the original release note - 

```
Improve queries that have unnecessary limits and order bys. This feature is enabled by default and can be disabled by using the configuration property optimizer.skip-redundant-sort or session property skip_redundant_sort.
```

So how about this - 

```
* Revert optimization to remove unnecessary ``LIMIT`` and ``ORDER BY`` (:pr:14915) since it caused incorrect results.
```